README.md

```
@@ -3,166 +3,237 @@
+// Delete all preference data
+RunCache.delete("user:*:preferences");
+
+// Check if any user 2 data exists
+const hasUser2Data = await RunCache.has("user:2:*");
+
+// Refresh all profile data
+await RunCache.refetch("user:*:profile");
+```
+
+Wildcard support is implemented for all key-based operations:
+- `get`: Returns an array of values for matching keys
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)